common/services/comments_service.py

```diff
@@ -0,0 +1,123 @@
+from typing import List
```

Comment:
The "service" part is just noise. Let's just create `comments.py` and `submits.py` under the `common` directory.